templates/osx_install.sh.erb

```diff
@@ -0,0 +1,9 @@
+function random_hexdump() {
+  hexdump -n 2 -e '/2 "%u"' /dev/urandom
+}
+
+mountpoint="$(mktemp -d -t $(random_hexdump))"
```

Comment:
We should be able to rely on mktemp to generate a securely random filename here - `$(mktemp -d)` is probably just fine on its own?

Reply:
Sure. For context, I ported this code over from https://github.com/puppetlabs/puppetlabs-puppet_agent/blob/master/tasks/install_shell.sh#L427